programs/voter-stake-registry/src/state/lockup.rs

```diff
@@ -147,6 +165,30 @@ mod tests {
     const MAX_SECS_LOCKED: u64 = 365 * 24 * 60 * 60 + 7 * 60 * 60;
     const MAX_DAYS_LOCKED: f64 = MAX_SECS_LOCKED as f64 / (24.0 * 60.0 * 60.0);
 
+    #[test]
+    pub fn period_computations() -> Result<()> {
+        let lockup = Lockup::new_from_periods(LockupKind::Daily, 1000, 3)?;
+        let day = SECS_PER_DAY as i64;
+        assert_eq!(lockup.periods_total()?, 3);
```

Comment:
can you add one comment what these asserts are testing? not obvious

Reply:
Hm, not sure what I should say here. It checks the math and boundary cases for `periods_total(), period_current(), periods_left()`